frontend/src/components/maps/MineMapLeaflet.js

```diff
@@ -176,19 +176,36 @@ const tenureLayerStyles = {
 };
 
 class MineMapLeaflet extends Component {
-  state = {
-    mapCreated: false,
-  };
-
   componentDidMount() {
+    // Create the basic leaflet map
+    this.map = L.map("leaflet-map", {
+      attributionControl: false,
+      center: [this.props.lat, this.props.long],
+      zoom: this.props.zoom,
+      worldCopyJump: true,
+      zoomAnimationThreshold: 8,
+      minZoom: 4,
+    });
```

Comment:
Confirming that this doesn't require anything fetched by `scriptLoader`? I'd be worried about a race condition here

Reply:
Pulled the branch down and tested this. All is well